docs/reference/commandline/info.md

```diff
@@ -55,7 +55,12 @@ information about the `devicemapper` storage driver is shown:
 
 ```bash
 $ docker info
+Client
+------
+Debug mode: false
```

Comment:
Looks like the code uses capital `M` now

Reply:
Indeed, that seems to be a longstanding difference rather than something I've done. Will fix it up since I'm touch it.